pkg/sql/codegen/tensorflow/codegen.go

```diff
@@ -215,7 +215,7 @@ func Train(trainIR *ir.TrainClause) (string, error) {
 	paiTable := ""
 	// TODO(typhoonzero): if isPAI, create two Couler steps
 	if isPAI {
-		fromRegex, err := regexp.Compile("FROM\\s([a-z0-9_\\.]*)")
+		fromRegex, err := regexp.Compile("FROM[\\s\\n]+([\\w\\.]*)")
```

Comment:
I'm not sure, but if my memory serves me, the golang `regexp` requires a `(?m)` prefix to enable the multiline mode, in that case, `\s` will match `\n` automatically. Can we add a test case for the multiline case?

Reply:
Already tested on http://regex-golang.appspot.com/